package.json

```diff
@@ -25,11 +25,7 @@
   "peerDependencies": {
     "@theforeman/vendor": ">= 4.0.0"
   },
-  "dependencies": {
-    "react-intl": "^2.8.0"
-  },
   "devDependencies": {
-    "@babel/core": "^7.7.0",
     "@sheerun/mutationobserver-shim": "^0.3.3",
```

Comment:
What is this one?

Reply:
It is referenced in `webpack/test_setup.js`.
After removing the code there, though the tests still run through.

So I guess it is obsolete :shrug: